KBaseSearchEngine.spec

```diff
@@ -130,13 +130,16 @@ module KBaseSearchEngine {
           ('key_props' field in ObjectData structure),
       skip_data - do not include raw data for object ('data' and 
           'parent_data' fields in ObjectData structure),
+      include_highlight - include highlights of fields that
+           matched query,
       ids_only - shortcut to mark all three skips as true.
```

Comment:
...and highlight as false.

Reply:
??

Reply:
ids_only description is no longer accurate.